.github/workflows/test.yml

```diff
@@ -91,13 +91,13 @@ jobs:
     - name: Wait for Opensearch
       # ensure that OS has come up and is available
       run: |
-        ./script/wait-for-endpoint --timeout=20 http://admin:admin@localhost:9200
+        ./script/wait-for-endpoint --timeout=20 http://admin:myStrongPassword123!@localhost:9200
     - name: Dump docker logs on failure
       if: failure()
       uses: jwalton/gh-docker-logs@v2
     - name: Run the tests
       run: |
-        export OPENSEARCH_URL=http://admin:admin@localhost:9200
+        export OPENSEARCH_URL=http://admin:myStrongPassword123!@localhost:9200
```

Comment:
an alternative approach: we can define an env variable for this action and use that.

```
env:
    ADMIN_PASSWORD: myStrongPassword123!
```
and use it as ${{ env.ADMIN_PASSWORD }}:

```
export OPENSEARCH_URL=http://admin:${{ env.ADMIN_PASSWORD }@localhost:9200
```